HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -211,12 +211,54 @@ def customiseForOffline(process):
     return process
 
 
+def customizeHLTfor38761(process):
+
+     for producer in producers_by_type(process, "SiPixelRecHitSoAFromLegacy"):
+         producer._TypedParameterizable__type = "SiPixelRecHitSoAFromLegacyPhase1"
+         if hasattr(producer, "isPhase2"):
+             delattr(producer, "isPhase2")
+     for producer in producers_by_type(process, "SiPixelDigisClustersFromSoA"):
+         producer._TypedParameterizable__type = "SiPixelDigisClustersFromSoAPhase1"
+         if hasattr(producer, "isPhase2"):
+             delattr(producer, "isPhase2")
+     for producer in producers_by_type(process, "SiPixelDigisClustersFromSoA"):
+         if hasattr(producer, "isPhase2"):
+             delattr(producer, "isPhase2")
+     if 'hltSiPixelRecHitsSoA' in process.__dict__:
+         process.hltSiPixelRecHitsSoA.cpu =  cms.EDAlias(hltSiPixelRecHitsFromLegacy = cms.VPSet(
+            cms.PSet(
+                type = cms.string('pixelTopologyPhase1TrackingRecHit2DCPUT')
+            ),
+            cms.PSet(
+                type = cms.string('uintAsHostProduct')
+            )))
+
+     producer_tochange =  ["SiPixelRecHitCUDA","SiPixelRecHitFromCUDA","SiPixelRecHitSoAFromCUDA","PixelTrackProducerFromSoA"]
+     producer_tochange = producer_tochange + ["PixelTrackDumpCUDA","PixelTrackSoAFromCUDA","CAHitNtupletCUDA"]
+     producer_tochange = producer_tochange + ["PixelVertexProducerCUDA","SeedProducerFromSoA"]
+
+     for thiproducer in producer_tochange:
+         for producer in producers_by_type(process, thiproducer):
+             producer._TypedParameterizable__type = thiproducer + "Phase1"
+
+     for producer in esproducers_by_type(process, "PixelCPEFastESProducer"):
+         producer._TypedParameterizable__type = "PixelCPEFastESProducerPhase1"
+         if hasattr(producer, "isPhase2"):
+             delattr(producer, "isPhase2")
+     for producer in esproducers_by_type(process, "PixelCPEGenericESProducer"):
+         if hasattr(producer, "Upgrade"):
+             setattr(producer,"isPhase2",getattr(producer,"Upgrade"))
+             delattr(producer, "Upgrade")
+
+     return process
+
```

Comment:
In order to do a migration of the HLT menu in ConfDb from the old plugins (without ```Phase1```) to the new plugins (with ```Phase1```), we would need one release in which BOTH are available (ie, both with cfi files). So could you please make one PR to add the new plugins, and then later another PR to remove the old plugins? Thanks!

Reply:
In the HLT customisation, there are TWO blocks with 
```
for producer in producers_by_type(process, "SiPixelDigisClustersFromSoA"):
```
is that a cut&paste error or could the two blocks be combined into one?

Reply:
@Martin-Grunewald did this get resolved?

Reply:
Hi @rappoccio this was referring to this duplication [here](https://github.com/cms-sw/cmssw/blob/d60eeb91b444e002df1a6bdc23b7b6b4e31b924c/HLTrigger/Configuration/python/customizeHLTforCMSSW.py#L243) (lines 245 and 247) solved in the commit [here](https://github.com/cms-sw/cmssw/commit/7c60fb9ed08a4c7ad2833c8ee3d3868bfb169fa2) (see under `HLTrigger/Configuration/python/customizeHLTforCMSSW.py`).